rlclientlib/schema/DecisionMode.fbs

```diff
@@ -0,0 +1,16 @@
+namespace reinforcement_learning.messages.flatbuff;
+
+union Mode { DefaultMode, ImitationMode }
+
+table DefaultMode {

```

Comment:
I believe we can find something more descriptive than `DefaultMode`.

Reply:
Fixed. Using OnlineMode